packages/react-storage/src/components/StorageBrowser/controls/ActionStartControl.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import { ControlProps } from './types';
+import { useResolvedComposable } from './hooks/useResolvedComposable';
+import { ActionStart } from '../composables/ActionStart';
+import { useActionStart } from './hooks/useActionStart';
+
+export const ActionStartControl = ({
+  className,
+}: ControlProps): React.JSX.Element | null => {
+  const { props } = useActionStart();
+  const ResolvedActionStart = useResolvedComposable(ActionStart, 'ActionStart');
+  if (!props) {
+    return null;
+  }
+  const { onClick, disabled, label } = props;
+
+  return (
+    <ResolvedActionStart
+      onClick={onClick}
+      disabled={disabled}
+      label={label}
+      className={className}
+    />
```

Comment:
Nit for consistency - I started doing this since the goal is for the props coming out of the control hook to be directly mapped to the composable:
```suggestion
  return (
    <ResolvedActionStart {...props} />
```

Reply:
Yeah, I thought it is better to pass it in explicitly so as to not pass in props inadvertently since it could be customer's composable component.

Reply:
That's why we have TS right? 😁